gateway/api_definition_test.go

```diff
@@ -1364,13 +1364,14 @@ func TestAPISpec_SanitizeProxyPaths(t *testing.T) {
 }
 
 func TestEnforcedTimeout(t *testing.T) {
-	test.Flaky(t) // TODO TT-5222
+	//test.Flaky(t) // TODO TT-5222
```

Comment:
If it hasn't been resolved, don't comment it out. Running `go test -run=TestEnforcedTimeout -count=1 -v .` locally should work. Don't touch if it's not part of the ticket.